main.c

```diff
@@ -26,6 +26,7 @@ void usage_main() {
 }
 
 int main(int argc, char *argv[]) {
+    //putc((int)('.'), stderr);
```

Comment:
Please remove

Reply:
Thanks, I thought I removed all of these, guess I missed some.